### PR TITLE
Stabilize four storage functions

### PR DESCRIPTION
**Pull request by agryaznov**
Follow-up to [substrate#12611](https://github.com/paritytech/substrate/pull/12611)

Stabilize the following functions:

- `[seal2] set_storage`
- `[seal1] get_storage`
- `[seal1] contains_storage`
- `[seal1] clear_storage`

### CODE REVIEW

**Review by athei — APPROVED** [review]
CI is expected to fail. We need to merge https://github.com/paritytech/substrate/pull/12611 and wait for a day.

**Review by xermicus — APPROVED** [review]

**Comment by agryaznov** [comment]
e2e tests [fail](https://gitlab.parity.io/parity/mirrors/ink/-/jobs/2017607) because [substrate-contracts-node](https://github.com/paritytech/substrate-contracts-node) sticks to [polkadot-v0.9.29](https://github.com/paritytech/substrate/tree/polkadot-v0.9.29) branch which does not have the funcs stabilized hence its getting to `CodeRejected: module imports a non-existent function`

**Comment by athei** [comment]
Bad news. `polkadot-v0.9.32` was branched off not today but at a much earlier commit. Dunno why they said "get your stuff in before wednesday". Maybe there will be another branch today. Otherwise we cherry-pick the stabilization onto the `polkadot-v0.9.32` release  branch.

**Comment by ascjones** [comment]
> Bad news. `polkadot-v0.9.32` was branched off not today but at a much earlier commit. Dunno why they said "get your stuff in before wednesday". Maybe there will be another branch today. Otherwise we cherry-pick the stabilization onto the `polkadot-v0.9.32` release branch.

https://github.com/paritytech/cargo-contract/pull/816

**Comment by athei** [comment]
[Published](https://github.com/paritytech/substrate-contracts-node/releases/tag/v0.22.0) a new `substrate-contracts-node` release which contains the stabilized functions.

File: crates/env/src/engine/on_chain/ext.rs [hunk 1]
```diff
@@ -430,6 +430,27 @@ mod sys {
...
+        // - `key_ptr`: pointer into the linear memory where the location to store the value is placed.
+        // - `key_len`: the length of the key in bytes.
+        // - `value_ptr`: pointer into the linear memory where the value to set is placed.
+        // - `value_len`: the length of the value in bytes.
+        //
+        // # Return Value
+        //
+        // Returns the size of the pre-existing value at the specified key if any. Otherwise
+        // `SENTINEL` is returned as a sentinel value.
+        pub fn seal_set_storage(
```
Comment (ascjones) [inline crates/env/src/engine/on_chain/ext.rs:447]:
```suggestion
        pub fn set_storage(
```
I've just merged https://github.com/paritytech/ink/pull/1482, so `seal` prefix should be removed?

**Pull request merged**